src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Alerts and AI transition guide 2022
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - EOL
+  - Transition guide
+metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+---
+
+Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
+
+## What's changing [#changes]
+
+Here is a summary of what we plan to EOL in 2022, why this is happening, and what will replace the current experience for Alerts and Applied Intelligence.
+
+If you have questions or comments about anything in this EOL transition guide, don't hesitate to:
+
+* Reach out to your New Relic account representative.
+* Use the reply thread in our [Explorers Hub post](https://discuss.newrelic.com/t/big-news-for-alerts-and-applied-intelligence/161436).
+
+<CollapserGroup>
+
+  <Collapser
+    id="outlier"
+    title="February 2022: NRQL baselines to replace outlier NRQL alert condition types."
+  >
+
+  **Reason for EOL:** Inefficient cluster identification and analysis capability.
+
+  **Replacement:** Use NRQL baselines to watch for anomalies in your cluster or to group similar signals. This method is an improvement over outlier conditions, since it will alert you to exactly which member of the group is misbehaving. You may be able to meet the same need as your old outlier conditions met by using a faceted baseline condition.
+
+  **Learn more:**
+
+  * [Faceted baseline conditions documentation](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-baseline-alert-conditions/#baseline-facets)
+  * [Explorers Hub](https://discuss.newrelic.com/t/nrql-outlier-alert-conditions-end-of-life/164167)
+
+  </Collapser>
+
+  <Collapser
+    id="query-thresholds"
+    title="May 2022: Sliding window aggregations to replace sum of query results thresholds."
+  >
+
+  **Reason for EOL:** This has not been an ideal way to address issues caused by sporadic signals. It lacks the ability to smooth out volatile signals or efficiently deal with gaps in signal.
```

Comment:
```suggestion
  **Reason for EOL:** [Sliding window aggregation](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/create-smoother-charts-sliding-windows) has not been an ideal way to address issues caused by sporadic signals. It lacks the ability to smooth out volatile signals or efficiently deal with gaps in signal.
```

Reply:
Just a general note that I think it might be good to include a link to the feature in question somewhere for all of these features to make sure they can realize quickly what feature we're talking about.

Reply:
That's a good point. I had left xrefs out of this draft because I wanted to steer them away from using these things anymore but your fresh eyes made me rethink this. Thanks!